server/methods/core/cart.js

```diff
@@ -702,6 +743,65 @@ Meteor.methods({
   },
 
   /**
+   * cart/setUserCurrency
+   * @summary saves user currency in cart, to be paired with order/setCurrencyExhange
+   * @param {String} cartId - cartId to apply setUserCurrency
+   * @param {String} userCurrency - userCurrency to set to cart
+   * @return {Number} update result
+   */
+  "cart/setUserCurrency": function (cartId, userCurrency) {
+    check(cartId, String);
+    check(userCurrency, String);
+    const cart = Collections.Cart.findOne({
+      _id: cartId,
+      userId: this.userId
+    });
+    if (!cart) {
+      Logger.error(`Cart not found for user: ${ this.userId }`);
```

Comment:
I'm seeing this error as `Cart for not found user: null` in the logs a few times. Seems like this should not normally happen?